api/build/compile_publish.go

```diff
@@ -320,12 +320,11 @@ func CompileAndPublish(
 				logrus.Errorf("unable to set commit status for %s/%d: %v", repo.GetFullName(), b.GetNumber(), err)
 			}
 
-			return false,
-				nil,
+			return nil,
 				&types.Item{
 					Build: b,
 				},
-				nil
+				fmt.Errorf(skip)
```

Comment:
should we use `errors.New(skip)` ?

im surprised golangci lets you use a format func without a format string

Reply:
Cool with me